zipkin-storage/cassandra-v1/src/main/java/zipkin2/storage/cassandra/v1/CassandraSpanConsumer.java

```diff
@@ -80,7 +84,9 @@
         if (span.name() == null) continue;
         insertSpanNames.add(insertSpanName.newInput(serviceName, span.name()));
       }
-
+      if (null != v2.tags() && !v2.tags().isEmpty()) {
```

Comment:
tags wont be null. I would also expect this to filter key names that actually should be indexed